2d/include/pcl/2d/impl/edge.hpp

```diff
@@ -42,439 +42,429 @@
   const int height = input_x.height;
   const int width = input_x.width;
 
-  output.resize (height * width);
+  output.resize(height * width);
   output.height = height;
   output.width = width;
 
   // Noise reduction using gaussian blurring
-  pcl::PointCloud<PointXYZI>::Ptr gaussian_kernel (new pcl::PointCloud<PointXYZI>);
-  kernel_.setKernelSize (3);
-  kernel_.setKernelSigma (1.0);
-  kernel_.setKernelType (kernel<PointXYZI>::GAUSSIAN);
-  kernel_.fetchKernel (*gaussian_kernel);
-  convolution_.setKernel (*gaussian_kernel);
+  pcl::PointCloud<PointXYZI>::Ptr gaussian_kernel(new pcl::PointCloud<PointXYZI>);
+  kernel_.setKernelSize(3);
+  kernel_.setKernelSigma(1.0);
+  kernel_.setKernelType(kernel<PointXYZI>::GAUSSIAN);
+  kernel_.fetchKernel(*gaussian_kernel);
+  convolution_.setKernel(*gaussian_kernel);
 
   PointCloudIn smoothed_cloud_x;
-  convolution_.setInputCloud (input_x.makeShared());
-  convolution_.filter (smoothed_cloud_x);
+  convolution_.setInputCloud(input_x.makeShared());
+  convolution_.filter(smoothed_cloud_x);
 
   PointCloudIn smoothed_cloud_y;
-  convolution_.setInputCloud (input_y.makeShared());
-  convolution_.filter (smoothed_cloud_y);
-
+  convolution_.setInputCloud(input_y.makeShared());
+  convolution_.filter(smoothed_cloud_y);
 
   // Edge detection using Sobel
-  pcl::PointCloud<PointXYZIEdge>::Ptr edges (new pcl::PointCloud<PointXYZIEdge>);
-  sobelMagnitudeDirection (smoothed_cloud_x, smoothed_cloud_y, *edges.get ());
+  pcl::PointCloud<PointXYZIEdge>::Ptr edges(new pcl::PointCloud<PointXYZIEdge>);
+  sobelMagnitudeDirection(smoothed_cloud_x, smoothed_cloud_y, *edges.get());
 
   // Edge discretization
-  discretizeAngles (*edges);
+  discretizeAngles(*edges);
 
-  pcl::PointCloud<PointXYZI>::Ptr maxima (new pcl::PointCloud<PointXYZI>);
-  suppressNonMaxima (*edges, *maxima, tLow);
+  pcl::PointCloud<PointXYZI>::Ptr maxima(new pcl::PointCloud<PointXYZI>);
+  suppressNonMaxima(*edges, *maxima, tLow);
 
   // Edge tracing
-  for (int i = 0; i < height; i++)
-  {
-    for (int j = 0; j < width; j++)
-    {
-      if ((*maxima)(j, i).intensity < tHigh || (*maxima)(j, i).intensity == std::numeric_limits<float>::max ())
+  for (int i = 0; i < height; i++) {
+    for (int j = 0; j < width; j++) {
+      if ((*maxima)(j, i).intensity < tHigh ||
+          (*maxima)(j, i).intensity == std::numeric_limits<float>::max())
         continue;
 
-      (*maxima)(j, i).intensity = std::numeric_limits<float>::max ();
-      cannyTraceEdge ( 1, 0, i, j, *maxima);
-      cannyTraceEdge (-1, 0, i, j, *maxima);
-      cannyTraceEdge ( 1, 1, i, j, *maxima);
-      cannyTraceEdge (-1, -1, i, j, *maxima);
-      cannyTraceEdge ( 0, -1, i, j, *maxima);
-      cannyTraceEdge ( 0, 1, i, j, *maxima);
-      cannyTraceEdge (-1, 1, i, j, *maxima);
-      cannyTraceEdge ( 1, -1, i, j, *maxima);
+      (*maxima)(j, i).intensity = std::numeric_limits<float>::max();
+      cannyTraceEdge(1, 0, i, j, *maxima);
```

Comment:
This might be one of the places to disable clang-format for the table like manual formatting

Reply:
Good point